code/defines/obj/weapon.dm

```diff
@@ -520,6 +520,13 @@
 	throw_speed = 1
 	throw_range = 3
 
+/obj/item/weapon/batteringram/attackby(var/obj/item/I, mob/user as mob)
+	if(istype(I,/obj/item/weapon/ram_kit))
+		flags = FPRINT | TWOHANDABLE //Loses MUSTTWOHAND
```

Comment:
flags &= MUSTTWOHAND would remove the MUSTTWOHAND flag, rather than manually resetting the flags.

Reply:
This is a good idea 👌

Reply:
`&= ~MUSTTWOHAND` that is